src/js/burials/containers/ConfirmationPage.jsx

```diff
@@ -83,7 +83,7 @@ class ConfirmationPage extends React.Component {
             </li>
           </ul>
         </div>
-        <p>Need help? If you have questions, call <a href="tel:800-827-1000">800-827-1000</a> from 8:00 a.m. - 9:00 p.m. EST Mon - Fri and have the Veteran’s Social security number or VA file number ready.</p>
+        <p>Need help? If you have questions, call <a href="tel:800-827-1000">800-827-1000</a> from 8:00 a.m. - 9:00 p.m. EST Mon - Fri and have the Veteran’s Social security number or VA file number ready. For telecommunication relay services, dial <a href="tel:711">711</a>.</p>
```

Comment:
@jbalboni I've made changes to include the `+1` on my local copy of this branch, please let me know if you'd like me to push them up.